src/Microsoft.ML.Data/Transforms/TransformsExtensionsCatalog.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    using HashDefaults = Transforms.HashEstimator.Defaults;
+
+    /// <summary>
+    /// Extensions for Column Copying Estiamtor.
+    /// </summary>
+    public static class ColumnCopyingCatalog
```

Comment:
>ColumnCopyingCatalog [](start = 24, length = 20)

What's the downside in merging it all in one class?

Reply:
We'd have files with just one method, if we didn't. 
And this might force us to keep thinking about categorizing them.  

---
In reply to: [229379681](https://github.com/dotnet/machinelearning/pull/1448#discussion_r229379681) [](ancestors = 229379681)